src/vl-select.js

```diff
@@ -24,17 +24,32 @@ Promise.all([
 * @property {boolean} data-vl-select-search-result-limit - Attribuut om het aantal resultaten te limiteren.
 * @property {boolean} data-vl-select-search-no-result-limit - Attribuut om het aantal resultaten te limiteren.
 * @property {boolean} data-vl-select-deletable - Attribuut om te activeren of deactiveren dat het geselecteerde kan verwijderd worden.
+* @property {string} data-vl-search-placeholder - Attribuut bepaalt de placeholder van het zoek adres input element.
+* @property {string} data-vl-search-no-results-text - Attribuut bepaalt de tekst wanneer er geen zoekresultaten meer zijn.
+* @property {string} data-vl-no-more-options - Attribuut bepaalt de tekst wanneer er geen keuzes meer mogelijk zijn.
 *
 * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-select/releases/latest|Release notes}
 * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-select/issues|Issues}
 * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-select.html|Demo}
 */
 export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelectElement)) {
+  static get DEFAULT_SEARCH_PLACEHOLDER() {
+    return 'Zoek item';
+  }
+
+  static get DEFAULT_SEARCH_NO_RESULT() {
+    return 'Geen resultaten gevonden';
+  }
+
+  static get DEFAULT_NO_MORE_OPTIONS() {
+    return 'Geen resterende opties gevonden';
+  }
```

Comment:
is het een probleem dat wij nu de defaults bepalen en niet meer de code in lib/select.js ?

Reply:
ik bedoel, als morgen de lib/select.js verandert, gaan wij toch deze strings er in steken als ge niets expliciet mee geeft, terwijl ze in lib/select.js misschien andere "defaults" hebben genomen.

Reply:
Momenteel zijn dezelfde teksten genomen als in de vl-ui-core translations (i18n) nl.js.
Als die morgen wijzigen in die file, hebben wij inderdaad niet meer dezelfde defaults.

Reply:
Besproken met @wautergu. Terechte opmerking van @wautergu dat een clone niet gewenst is omdat er applicatief meerdere vl-select bestanden ingeladen kunnen worden waardoor een nieuwe clone potentieel gewijzigde teksten bevat. Best dus huidige implementatie behouden. Er werd een unit test toegevoegd om te controleren of de teksten overeenkomen met die dat centraal gedefinieerd werden.